File: pm2/twitter/lib/model/http_response_models/follow_result.dart
// To parse this JSON data, do
//
//     final followResult = followResultFromJson(jsonString);

import 'dart:convert';

FollowResult followResultFromJson(String str) =>
    FollowResult.fromJson(json.decode(str));

String followResultToJson(FollowResult data) => json.encode(data.toJson());

class FollowResult {
  String follower;
  String following;
  bool follows;

  FollowResult({
    this.follower,
    this.following,
    this.follows,
  });

  factory FollowResult.fromJson(Map<String, dynamic> json) => FollowResult(
        follower: json["follower"],
        following: json["following"],
        follows: json["follows"],
      );

  Map<String, dynamic> toJson() => {
        "follower": follower,
        "following": following,
        "follows": follows,
      };
}

File: pm3/twitter-backend/src/lambdas/user/actions/follow/FollowResult.java
package lambdas.user.actions.follow;

public class FollowResult {
    public String follower;
    public String following;
    public boolean follows;

    public FollowResult(String follower, String following, boolean follows) {
        this.follower = follower;
        this.following = following;
        this.follows = follows;
    }
}
